modules/cats/src/test/scala/pureconfig/module/cats/CatsSuite.scala

```diff
@@ -0,0 +1,23 @@
+package pureconfig.module.cats
+
+import cats.data.{ NonEmptyList, NonEmptyVector }
+import com.typesafe.config.ConfigFactory
+import org.scalatest._
+import pureconfig.syntax._
+
+class CatsSuite extends FlatSpec with Matchers with EitherValues {
+
+  case class Numbers(numbers: NonEmptyList[Int])
+
+  it should "be able to read a config with a NonEmptyList" in {
```

Comment:
Can you add tests for checking that reading empty lists and vectors return a `EmptyTraversableFound` failure?